resources/migrations/001_update_migrations.yaml

```diff
@@ -5205,6 +5205,40 @@ databaseChangeLog:
         - dbms:
             type: mysql,mariadb
 
+  - changeSet:
+      id: v49.2024-01-22T11:50:00
+      author: qnkhuat
+      comment: Add `report_card.type`
+      changes:
+        - addColumn:
+            tableName: report_card
+            columns:
+              - column:
+                  remarks: The type of card, could be 'question', 'model', 'metric'
+                  name: type
```

Comment:
I think you should add:
```
constraints:
  nullable: false
```

Reply:
by default the column will not be nullable. but yea let's add it for clarity